docs/components/102-commendation-vendor.rst

```diff
@@ -6,5 +6,4 @@ lost or deleted faction items for faction tokens. The only
 object this is attached to is
 :lot:`Honor Accolade -- Commendation Vendor <13806>`.
 
-More information on him may be found here:
-`Honor Accolade <http://legouniverse.wikia.com/wiki/Honor_Accolade>`_
+More information on may be found here: :wiki:`Honor Accolade <Honor_Accolade>`
```

Comment:
Note: we could probably auto-normalize spaces to underscores in these cases

Reply:
How would I go about doing this?  just .replace("_", " ") ?

Reply:
The other way around, but yeah. Just `.replace(" ", "_") to turn a title into the url suffix.